_tools/check_browser_compat.ts

```diff
@@ -6,11 +6,14 @@
  *
  * Run using: deno run --allow-read --allow-run _tools/check_browser_compat.ts
  */
-import { walkSync } from "../fs/walk.ts";
+
+import { walk } from "../fs/walk.ts";
+import { COPYRIGHT } from "./check_licence.ts";
 
 const ROOT = new URL("../", import.meta.url);
 const SKIP = [/(test|bench|\/_|\\_|testdata|version.ts)/];
 const DECLARATION = "// This module is browser compatible.";
```

Comment:
Not directly related to this PR, but some file (`data_structures/comparators.ts`) seems having `/** This module is browser compatible. */` comment

Reply:
Ok, I guess we should replace that with a single line comment in another PR.